app/uk/gov/hmrc/cgtpropertydisposals/models/returns/CompleteReturn.scala

```diff
@@ -32,7 +33,8 @@ final case class CompleteReturn(
   acquisitionDetails: CompleteAcquisitionDetailsAnswers,
   reliefDetails: CompleteReliefDetailsAnswers,
   exemptionsAndLossesDetails: CompleteExemptionAndLossesAnswers,
-  yearToDateLiabilityAnswers: CompleteCalculatedYearToDateLiabilityAnswers
+  yearToDateLiabilityAnswers: CompleteCalculatedYearToDateLiabilityAnswers,
+  initialGainAnswers: Option[AmountInPence]
```

Comment:
`initialGainOrLoss` rather than `initialGain` would be a better name here so that it's clear this value can include a loss as well

Reply:
it's been renamed